packages/mdc-chips/README.md

```diff
@@ -153,7 +177,7 @@ CSS Class | Description
 
 ### Sass Mixins
 
-To customize the colors of any part of the chip, use the following mixins.
+To customize a chip, use the following mixins.
```

Comment:
We could drop this paragraph, esp. since the first mixin applies to the chip set.